selfie-camera/src/main/java/org/odk/collect/selfiecamera/CameraXCamera.kt

```diff
@@ -57,4 +74,35 @@ class CameraXCamera : Camera {
             }
         )
     }
+
+    @SuppressLint("MissingPermission")
+    override fun startVideo(
```

Comment:
I ended up doing more work on the video code - the code felt in a really incomplete/inconsistent place and it didn't feel like much more work to get it to a better place.